.travis.yml

```diff
@@ -37,4 +37,3 @@ addons:
     packages:
       - xvfb
       - g++-4.8
-      - libgnome-keyring-dev
```

Comment:
gnome-keyring is needed at runtime by muon, not sure if removing this will cause issues. 
see: https://github.com/brave/browser-laptop/issues/10448